docker-for-azure/upgrade.md

```diff
@@ -42,6 +42,18 @@ $ docker run \
   docker4x/upgrade-azure:17.06.0-ce-azure1
 ```
 
+If you are already on a version more recent than 17.06.1 CE or 17.07.0 CE, you can use the below script to initiate the upgrade:
+
+```bash
+$ upgrade.sh YY.MM.X-ce-azureC
+```
+
+For example, to upgrade from 17.06.1 CE edge to 17.07.0 CE edge, you can use:
+
+```bash
+$ upgrade.sh 17.07.0-ce-azure1
```

Comment:
Getting the exact build may be difficult, since we don't advertise them here:
https://docs.docker.com/docker-for-azure/release-notes/
It would be ideal if we could do: `upgrade 17.07.0` and we figure out the latest version.

The script could then lookup the proper tag via:
`docker run --rm docker4x/awscli:latest s3api --no-sign-request get-object --bucket editions-us-east-1 --key azure/stable/17.06.0/Docker.tmpl docker.out | jq -r '.Metadata | .edition_version'`